src/testdir/test_highlight.vim

```diff
@@ -1086,8 +1086,8 @@ func Test_hlget()
   let lines =<< trim END
     call assert_notequal([], filter(hlget(), 'v:val.name == "Visual"'))
     call assert_equal([], hlget('SomeHLGroup'))
-    highlight MyHLGroup term=standout cterm=reverse ctermfg=10 ctermbg=Black
-    call assert_equal([{'id': hlID('MyHLGroup'), 'ctermfg': '10', 'name': 'MyHLGroup', 'term': {'standout': v:true}, 'ctermbg': '0', 'cterm': {'reverse': v:true}}], hlget('MyHLGroup'))
+    highlight MyHLGroup term=standout cterm=inverse ctermfg=10 ctermbg=Black
+    call assert_equal([{'id': hlID('MyHLGroup'), 'ctermfg': '10', 'name': 'MyHLGroup', 'term': {'standout': v:true}, 'ctermbg': '0', 'cterm': {'inverse': v:true}}], hlget('MyHLGroup'))
```

Comment:
This behavior change seems backward-incompatible.

Reply:
This comes about because both "reverse" and "inverse" share the same #define value (HL_INVERSE) and when I resorted the highlight table for searching, sequence of the two attributes was reversed. A fix is to use an index which restores the original sequence of highlight attributes for display purposes.